src/styles/styles.js

```diff
@@ -3257,6 +3261,27 @@ const styles = {
         textAlign: 'center',
     },
 
+    linkPreviewWrapper: {
+        marginTop: 16,
+        borderLeftWidth: 4,
+        borderLeftColor: getTransparentColor(themeColors.inverse, 33),
+        paddingLeft: 12,
+    },
+
+    linkPreviewImage: {
+        flex: 1,
+        resizeMode: 'contain',
+        maxWidth: 350,
+        maxHeight: 350,
```

Comment:
Good call - I think it's a good idea to make it something like: 
`maxHeight: Math.min(350, someValueBasedOnScreenSize)`
So it will be scaled down if the screen size of the phone is too small